tests/ci_install.sh

```diff
@@ -55,11 +57,10 @@ else
     # include a pyproject.toml file or use any pip-comptatible solution to remedy this.
     python -m pip install numpy>=1.19.4 cython~=0.29.21
 
-    # this is required for cartopy. It should normally be specified in our setup.cfg as
-    # cartopy[plotting]
-    # However it doesn't work on Ubuntu 18.04 (used in CI at the time of writing)
-    python -m pip install shapely --no-binary=shapely
```

Comment:
I remember that causing problems in envs different than 18.04 too. Is really worth dropping? This way we ensure that shapely builds against the same libraries as cartopy. Unless of course they've fixed all that mess upstream.

Reply:
Ah, good call I guess the comment isn't accurate enough then. I can revert this change.

I also note that we actually install cartopy only on Windows (via conda), so do you think this block should be moved to the windows-only part of the script ?

Reply:
I thought `[full]` brings it in?

Reply:
OK, but that script is also used in testing right? Cause otherwise why would we add it in the first place if nothing uses it?

Reply:
I suppose it may be a leftover from a time we _did_ try to have `cartopy` in `[full]`, but honestly not sure. I'm happy to do a pure, conservative revert if you think it's best.

for reference the problem seems to still be real, see
- https://github.com/SciTools/cartopy/issues/805
- https://github.com/SciTools/cartopy/issues/2006

Reply:
(I dropped this commit)